scripts/partition_manager/partition_manager.rst

```diff
@@ -190,7 +190,11 @@ share_size: list
    It cannot be used by phony partitions.
    The list can contain any kind of partition.
 
-   Note that if the target partition is the ``app`` or a partition that spans over the ``app``, the size is effectively split between them, because the size of the ``app`` is dynamically decided.
+   If the target partition is the ``app`` or a partition that spans over the ``app``, the size is effectively split between them, because the size of the ``app`` is dynamically decided.
+
+   If non of the partitions in the ``share_size`` list exists, and the partition does not define a ``size`` property, then the partition is removed.
```

Comment:
s/non/none/

Reply:
Hm, looks like this is fixed in a later commit.